flow/src/lib.rs

```diff
@@ -535,24 +552,24 @@ pub mod pallet {
 			origin: OriginFor<T>,
 			org_id: T::Hash,
 			admin_id: T::AccountId,
-			name: Vec<u8>,
+			name: BoundedVec<u8, T::StringLimit>,
 			target: T::Balance,
 			deposit: T::Balance,
 			expiry: T::BlockNumber,
 			protocol: FlowProtocol,
 			governance: FlowGovernance,
-			cid: Vec<u8>,
-			token_symbol: Vec<u8>, // up to 5
-			token_name: Vec<u8>,   /* cleartext
+			cid: BoundedVec<u8, T::StringLimit>,
+			token_symbol: BoundedVec<u8, T::StringLimit>, // up to 5
+			token_name: BoundedVec<u8, T::StringLimit>,   /* cleartext
 									* token_curve_a: u8,	  // preset
-									* token_curve_b: Vec<u8>, // custom */
+									* token_curve_b: BoundedVec<u8, T::StringLimit>, // custom */
 		) -> DispatchResultWithPostInfo {
 			let creator = ensure_signed(origin)?;
-			let owner = T::Control::org_controller_account(&org_id);
+			let owner = T::Control::org_controller_account(&org_id).ok_or(Error::<T>::InvalidId)?;
```

Comment:
Could be the same Error as in L675, bu timho either both are invalid or unknown, correct?

Reply:
Makes sense to return another error here. We already have `OwnerUnknown` one, gonna use that here.